asb-ballerina/message.bal

```diff
@@ -33,7 +33,7 @@
 @display {label: "Message"}
 public type Message record {|
     @display {label: "Body"}
-    string|xml|json|byte[] body;
+    string|int|byte[] body;
```

Comment:
Let's add a comment here stating that these are the basic AMQP message types. (point the reference)